boards/arm64/xenvm/xenvm-qemu.dtsi

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2023 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * This file was created by running
```

Comment:
This file was NOT created by LIBXL_DUMP_DTB, it is manually created for qemu Dom0. Please, remove or edit this comment.

Reply:
Done